client/my-sites/domains/domain-management/list/bulk-site-domains.tsx

```diff
@@ -82,6 +83,7 @@ export default function BulkSiteDomains( props: BulkSiteDomainsProps ) {
 	// If user has more than 1 domain on more than 1 site, show manage all domains CTA.
 	const { domains: allDomains } = useDomainsTable( fetchAllDomains );
 	const showManageAllDomainsCTA =
+		! isEnabled( 'layout/dotcom-nav-redesign' ) &&
```

Comment:
Do we also need to check whether the admin interface is classic as per the acceptance criteria in https://github.com/Automattic/dotcom-forge/issues/5774?

Reply:
@allilevine @ebinnion not sure why we're making this change.

Is it because global view is more prominent now?

Reply:
> Do we also need to check whether the admin interface is classic

I assume we don't as the top-level domain page is available regardless of the admin interface 🤔

Reply:
Oh yes, in that case just the nav redesign flag should be enough.